src/handler/http/request/search/mod.rs

```diff
@@ -214,6 +214,11 @@ pub async fn search(
             let user: meta::user::User =
                 USERS.get(&format!("{org_id}/{}", user_id)).unwrap().clone();
 
+            let stream_type = if stream_type.eq(&StreamType::Index) {
+                StreamType::Logs
+            } else {
+                stream_type
+            };
```

Comment:
@Subhra264 now we support created index stream for `logs`, `metrics` and `traces`, so maybe this is not working.

Reply:
Yes, will change it.